101-lustre-client-server-create/README.md

```diff
@@ -0,0 +1,18 @@
+﻿# Lustre client and server node VMs deployment
+
+<a href="https://portal.azure.com/#create/Microsoft.Template/uri/https://raw.githubusercontent.com/lustrehpc/azure-quickstart-templates/master/101-lustre-client-server-create/azuredeploy.json" target="_blank">
+<img src="http://azuredeploy.net/deploybutton.png"/>
+</a>
+<a href="http://armviz.io/#/?load=https://raw.githubusercontent.com/lustrehpc/azure-quickstart-templates/master/101-lustre-client-server-create/azuredeploy.json" target="_blank">
```

Comment:
Please let me know if it is mandatory to change lustre to "Lustre" as changing this will require a new repository to be created since the existing repository cannot be accessed.